packages/storybook/package.json

```
@@ -0,0 +1,47 @@
+{
+  "name": "@nrwl/storybook",
+  "version": "0.0.1",
+  "description": "Storybook plugin for Nx",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/nrwl/nx.git"
+  },
+  "keywords": [
+    "Monorepo",
+    "Angular",
+    "React",
+    "Web",
+    "Node",
+    "Nest",
+    "Jest",
+    "Storybook",
+    "Cypress",
+    "CLI"
+  ],
+  "main": "index.js",
+  "types": "index.d.ts",
+  "author": "Isaac Mann",
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/nrwl/nx/issues"
+  },
+  "homepage": "https://nx.dev",
+  "schematics": "./collection.json",
+  "builders": "./builders.json",
+  "ng-update": {
+    "requirements": {},
+    "migrations": "./migrations.json"
+  },
+  "peerDependencies": {
+    "@nrwl/workspace": "*"
+  },
+  "dependencies": {
+    "@angular-devkit/architect": "0.800.1",
+    "@angular-devkit/core": "8.0.1",
+    "@nrwl/cypress": "8.2.0",
```

```suggestion
    "@nrwl/cypress": "*",
```

This will get replaced with the same version of `@nrwl/cypress`